content/blog/release-2.4.0.md

```diff
@@ -0,0 +1,133 @@
+author: Ivan Komissarov
+---
+
+The [Qbs build tool](http://qbs.io) version 2.4.0 is available.
+
+## What's new
+
+The main feature in this release is the support for the [Conan](https://conan.io) package
+manager. We've added the new [conan](https://qbs.io//docs/qml-qbsmoduleproviders-conan/) module
+provider which creates Qbs modules from JSON files generated by Conan.
+
+<!--more-->
+
+### Conan Module provider
+
+The Conan module provider requires the `QbsDeps` generator which is present in Conan 2.5.0.
+For each dependency specified in the `conanfile`, this generator creates a JSON file with
+the information required for the conan module provider. This information includes paths to
+libraries, include directories, build tools, compiler and linker flags.
+Conan module provider reads those files and converts them to Qbs modules.
+
+Condider an example of an application that uses the `zlib` compression library:
+```
+CppApplication {
+    Depends { name: "zlib" }
+    qbsModuleProviders: "conan"
+    consoleApplication: true
+    files: "main.c"
+}
+```
+
+In order to install and build an application, we would also need a `conanfile.py`:
+```
+from conan import ConanFile
+from conan.tools.qbs import Qbs
+
+class Recipe(ConanFile):
+    name = "conan-zlib"
+    version = "1.0"
+
+    exports_sources = "*.c", "*.qbs",
+    settings = "os", "compiler", "arch", "build_type"
+    generators = ["QbsDeps"]
+
+    def build(self):
+        qbs = Qbs(self)
+        qbs.resolve()
+        qbs.build()
+```
+
+Now we can easily build the application using Conan:
+```
+$ conan create . --build=missing
+```
+
+Note that the new provider only works with Conan version 2.5.0 and above; for Conan version 1.x
+please use the [ConanfileProbe](http://localhost/~abbapoh/qbs-io//docs/qml-qbsprobes-conanfileprobe/).
+
+For more details on the provider as well as how to use Conan only for installing dependencies,
+see the [provider](https://qbs.io//docs/qml-qbsmoduleproviders-conan/) page in the documentation.
+
+### Other changes
+* We've added new FlatBuffers modules, for [C](https://qbs.io//docs/qml-qbsmodules-flatbuf-c/)
+  and [C++](https://qbs.io//docs/qml-qbsmodules-flatbuf-cpp/) languages. Those modules require
+  runtime dependencies that can be installed using Conan module provider
+  ([QBS-1666](https://bugreports.qt.io/projects/QBS/issues/QBS-1666)).
+* Rules trying to create artifacts outside the build directory is now a hard error
+  ([QBS-1268](https://bugreports.qt.io/projects/QBS/issues/QBS-1268)).
+* More details are now printed when a command times out
+  ([QBS-1750](https://bugreports.qt.io/projects/QBS/issues/QBS-1750)).
+* Updated the bundled quickjs library.
+* The pkg-config based fallback provider was removed. That provider was not flexible enough and
+  didn't give users control over providers priorities. Also, that provider created files
+  even for non-present modules which was inful in case of misconfiguration since those files
```

Comment:
"inful"?

Reply:
looks like I was distracted while rewording this sentence=)